crates/bench_utils/src/lib.rs

```diff
@@ -0,0 +1,188 @@
+mod utils;
+
+use proc_macro::TokenStream;
+use proc_macro2::{Ident, Span};
+use quote::quote;
+use utils::{
+  parse_arr_assign, parse_struct_assign, ArrayAssign, StructAssign, TreeLevel,
+};
+
+/// Repeatedly assigns to a variable to increase the number of locations
+/// while keeping the number of places constant.
```

Comment:
In each doc comment, it would be helpful to give a small example of a program generated by this macro.